File: single-language-analysis/dataset-files/java/TestSmell009.java
// Original URL: https://github.com/gabrielseibel1/MusicSeibuilder/blob/e931ba43f8edab3ec38006ef8476e77425726952/SoundTest.java#L78-L86

public class TestSmell009 {
    @Test
    public void silence() throws Exception {
        Sound sound = new Sound();
        sound.setNote('A');
        assertEquals('A',sound.getNote());

        sound.silence();
        assertEquals('R',sound.getNote());
    }
}

File: single-language-analysis/dataset-files/CSharp/CSharp.Tests/Tests/TestSmell009.cs
using Xunit;

public class TestSmell009
{
    [Fact]
    public void Silence()
    {
        Sound sound = new Sound();
        sound.SetNote('A');
        Assert.Equal('A', sound.GetNote());

        sound.Silence();
        Assert.Equal('R', sound.GetNote());
    }
}